Makefile

```diff
@@ -73,9 +73,13 @@ unittest:
 
 create-integration-config:
 ifneq ("${LINODE_TOKEN}", "")
-	@echo "api_token: ${LINODE_TOKEN}" > $(INTEGRATION_CONFIG);
+	@echo -n > $(INTEGRATION_CONFIG)
+	ansible-playbook scripts/create_e2e_cloud_firewall.yaml
+	@echo "api_token: ${LINODE_TOKEN}" >> $(INTEGRATION_CONFIG);
 else ifneq ("${LINODE_API_TOKEN}", "")
-	@echo "api_token: ${LINODE_API_TOKEN}" > $(INTEGRATION_CONFIG);
+	@echo -n > $(INTEGRATION_CONFIG)
+	ansible-playbook scripts/create_e2e_cloud_firewall.yaml
+	@echo "api_token: ${LINODE_API_TOKEN}" >> $(INTEGRATION_CONFIG);
 else
```

Comment:
Is there a way to do it within the test framework (`ansible-test`)?

Reply:
There are other ways to do it within the framework but the current design makes it very messy if we decide to take that route. E.g. Each test would require its own instance of the firewall, significantly increasing the amount of code in each file